cli/cli.go

```diff
@@ -25,6 +26,12 @@ func GetJfrogCliArtifactoryApp() components.App {
 		Commands:    evidenceCLI.GetCommands(),
 		Category:    "Command Namespaces",
 	})
+	app.Subcommands = append(app.Subcommands, components.Namespace{
+		Name:        string(cliutils.Rt),
+		Description: "Artifactory commands.",
+		Commands:    artifactoryCLI.GetCommands(),
```

Comment:
As I commented on the CLI PR, consider exposing these command directly for easier appending on the CLI side.